src/Microsoft.IIS.WebManager/app/files/file-nav.service.ts

```diff
@@ -16,6 +14,9 @@ import { ApiFile, ApiFileType, ChangeType } from './file';
 import { Navigator } from './navigator';
 import { FilesService } from './files.service';
 
+import 'rxjs/add/operator/pairwise'
```

Comment:
This change took me an whole afternoon to figure out
Static methods import need to be placed after the class it refer to is imported. Otherwise, there's no compile time error but at run time these methods would just go missing.